### DISCUSSION TITLE
OpenRadioss support on LS-DYNA card *contact_automatic_surface_to_surface_tiebreak

### DISCUSSION THREAD

**Discussion by halfwitbm** [OpenRadioss/OpenRadioss#1878] [Q&A] [answered]
Hi All,

CFG information in OpenRadioss code showed it support *contact_automatic_surface_to_surface_tiebreak  card. But on Altair Radioss help web, I can not find any information on it. Can someone give me some help on it? Thanks.

Another issue is when I try to output contact force in OpenRadioss, I find simulation results looks good, but the contact force output is always zero, as if it was not been outputted. If somebody has experience on this field, I can share my input deck. Please help me figure out. Thanks.

Bin

**Comment by PaulAltair** [comment]
*Contact_Automatic_Surface_To_Surface 'Tiebreak' option is not read/understood by OpenRadioss, it exists in CFG only to allow OpenRadioss to ignore the tiebreak and proceed with it as a normal surface-surface without giving an error to the user forcing them to change the input. The force based contact failure type doesn't really translate directly to any OpenRadioss feature

Regarding the contact forces, I see them ok in my models, if you can share yours I can take a look

**Reply by halfwitbm** [reply]
Thank you very much for your answer, Paul. I don't know how to share the input deck, so I paste it here. I just want to output contact force by OpenRadioss and see the result. Thanks.

Bin

$# LS-DYNA Keyword file created by LS-PrePost(R) V4.8.29 - 05Apr2022
$# Created on Sep-17-2023 (21:48:47)
*KEYWORD
*TITLE
$#                                                                         title
LS-DYNA keyword deck by LS-PrePost
*CONTROL_TERMINATION
$#  endtim    endcyc     dtmin    endeng    endmas     nosol     
      10.0         0       0.0       0.01.000000E8         0
*DATABASE_GLSTAT
$#      dt    binary      lcur     ioopt     
       0.1         2         0         1
*DATABASE_MATSUM
$#      dt    binary      lcur     ioopt     
       0.1         2         0         1
*DATABASE_RCFORC
$#      dt    binary      lcur     ioopt     
       0.1         2         0         1
*DATABASE_BINARY_D3PLOT
$#      dt      lcdt      beam     npltc    psetid      
       1.0         0         0         0         0
*BOUNDARY_PRESCRIBED_MOTION_SET_ID
$#      id                                                               heading
         0Disp
$#    nsid       dof       vad      lcid        sf       vid     death     birth
         2         3         2         1       1.0         01.00000E28       0.0
*BOUNDARY_SPC_SET
$#    nsid       cid      dofx      dofy      dofz     dofrx     dofry     dofrz
         1         0         1         1         1         1         1         1
*SET_NODE_LIST_TITLE
Fixed
$#     sid       da1       da2       da3       da4    solver      
         1       0.0       0.0       0.0       0.0MECH
$#    nid1      nid2      nid3      nid4      nid5      nid6      nid7      nid8
         1         2         3         4         5         6         7         8
         9        10        11        12        13        14        15        16
        17        18        19        20        21        22        23        24
        25        26        27        28        29        30        31        32
        33        34        35        36        37        38        39        40
        41        42        43        44        45        46        47        48
        49        50        51        52        53        54        55        56
        57        58        59        60        61        62        63        64
        65        66        67        68        69        70        71        72
        73        74        75        76        77        78        79        80
        81        82        83        84        85        86        87        88
        89        90        91        92        93        94        95        96
        97        98        99       100       101       102       103       104
       105       106       107       108       109       110       111       112
       113       114       115       116       117       118       119       120
       121       158       159       160       161       162       163       164
       165       166       167       168       169       170       171       172
       173       174       175       176       177       178       179       180
       181       182       183       184       185       186       187       188
       189       190       191       192       193       194       195       196
       197       198       199       200       201       202       203       204
       205       206       207       208       209       210       211       212
       213       214       215       216       217       218       219       220
       221       222       223       224       225       226       227       228
       229       230       231       232       233       234       235       236
       237       238       239       240       241       242       243       244
       245       246       247       248       249       250       251       252
       253       254       255       256       257       258       259       260
       261       262       263       264       265       266       267       268
       269       270       271       272       273       274       275       276
       277       278         0         0         0         0         0         0
*BOUNDARY_SPC_SET
$#    nsid       cid      dofx      dofy      dofz     dofrx     dofry     dofrz
         2         0         1         1         0         1         1         1
*SET_NODE_LIST_TITLE
Top
$#     sid       da1       da2       da3       da4    solver      
         2       0.0       0.0       0.0       0.0MECH
$#    nid1      nid2      nid3      nid4      nid5      nid6      nid7      nid8
       122       123       124       125       126       127       128       129
       130       131       132       133       134       135       136       137
       138       139       140       141       142       143       144       145
       146       147       148       149       150       151       152       153
       154       155       156       157       279       280       281       282
       283       284       285       286       287       288       289       290
       291       292       293       294       295       296       297       298
       299       300       301       302       303       304       305       306
       307       308       309       310       311       312       313       314
*PART
$#                                                                         title
Base 1
$#     pid     secid       mid     eosid      hgid      grav    adpopt      tmid
         1         1         1         0         0         0         0         0
*SECTION_SHELL
$#   secid    elform      shrf       nip     propt   qr/irid     icomp     setyp
         1        16       1.0         5       1.0         0         0         1
$#      t1        t2        t3        t4      nloc     marea      idof    edgset
       1.0       1.0       1.0       1.0       0.0       0.0       0.0         0
*MAT_ELASTIC_TITLE
steel
$#     mid        ro         e        pr        da        db  not used        
         17.90000E-6     210.0      0.33       0.0       0.0       0.0
*PART
$#                                                                         title
Base 2
$#     pid     secid       mid     eosid      hgid      grav    adpopt      tmid
         2         2         1         0         0         0         0         0
*SECTION_SHELL
$#   secid    elform      shrf       nip     propt   qr/irid     icomp     setyp
         2        16       1.0         5       1.0         0         0         1
$#      t1        t2        t3        t4      nloc     marea      idof    edgset
       1.0       1.0       1.0       1.0       0.0       0.0       0.0         0
*DEFINE_CURVE_TITLE
Disp
$#    lcid      sidr       sfa       sfo      offa      offo    dattyp     lcint
         1         0       1.0       1.0       0.0       0.0         0         0
$#                a1                  o1  
                 0.0                 0.0
                10.0                 1.0
*ELEMENT_SHELL
$#   eid     pid      n1      n2      n3      n4      n5      n6      n7      n8
       1       1       1       2      13      12       0       0       0       0
       2       1       2       3      14      13       0       0       0       0
       3       1       3       4      15      14       0       0       0       0
       4       1       4       5      16      15       0       0       0       0
       5       1       5       6      17      16       0       0       0       0
       6       1       6       7      18      17       0       0       0       0
       7       1       7       8      19      18       0       0       0       0
       8       1       8       9      20      19       0       0       0       0
       9       1       9      10      21      20       0       0       0       0
      10       1      10      11      22      21       0       0       0       0
      11       1      12      13      24      23       0       0       0       0
      12       1      13      14      25      24       0       0       0       0
      13       1      14      15      26      25       0       0       0       0
      14       1      15      16      27      26       0       0       0       0
      15       1      16      17      28      27       0       0       0       0
      16       1      17      18      29      28       0       0       0       0
      17       1      18      19      30      29       0       0       0       0
      18       1      19      20      31      30       0       0       0       0
      19       1      20      21      32      31       0       0       0       0
      20       1      21      22      33      32       0       0       0       0
      21       1      23      24      35      34       0       0       0       0
      22       1      24      25      36      35       0       0       0       0
      23       1      25      26      37      36       0       0       0       0
      24       1      26      27      38      37       0       0       0       0
      25       1      27      28      39      38       0       0       0       0
      26       1      28      29      40      39       0       0       0       0
      27       1      29      30      41      40       0       0       0       0
      28       1      30      31      42      41       0       0       0       0
      29       1      31      32      43      42       0       0       0       0
      30       1      32      33      44      43       0       0       0       0
      31       1      34      35      46      45       0       0       0       0
      32       1      35      36      47      46       0       0       0       0
      33       1      36      37      48      47       0       0       0       0
      34       1      37      38      49      48       0       0       0       0
      35       1      38      39      50      49       0       0       0       0
      36       1      39      40      51      50       0       0       0       0
      37       1      40      41      52      51       0       0       0       0
      38       1      41      42      53      52       0       0       0       0
      39       1      42      43      54      53       0       0       0       0
      40       1      43      44      55      54       0       0       0       0
      41       1      45      46      57      56       0       0       0       0
      42       1      46      47      58      57       0       0       0       0
      43       1      47      48      59      58       0       0       0       0
      44       1      48      49      60      59       0       0       0       0
      45       1      49      50      61      60       0       0       0       0
      46       1      50      51      62      61       0       0       0       0
      47       1      51      52      63      62       0       0       0       0
      48       1      52      53      64      63       0       0       0       0
      49       1      53      54      65      64       0       0       0       0
      50       1      54      55      66      65       0       0       0       0
      51       1      56      57      68      67       0       0       0       0
      52       1      57      58      69      68       0       0       0       0
      53       1      58      59      70      69       0       0       0       0
      54       1      59      60      71      70       0       0       0       0
      55       1      60      61      72      71       0       0       0       0
      56       1      61      62      73      72       0       0       0       0
      57       1      62      63      74      73       0       0       0       0
      58       1      63      64      75      74       0       0       0       0
      59       1      64      65      76      75       0       0       0       0
      60       1      65      66      77      76       0       0       0       0
      61       1      67      68      79      78       0       0       0       0
      62       1      68      69      80      79       0       0       0       0
      63       1      69      70      81      80       0       0       0       0
      64       1      70      71      82      81       0       0       0       0
      65       1      71      72      83      82       0       0       0       0
      66       1      72      73      84      83       0       0       0       0
      67       1      73      74      85      84       0       0       0       0
      68       1      74      75      86      85       0       0       0       0
      69       1      75      76      87      86       0       0       0       0
      70       1      76      77      88      87       0       0       0       0
      71       1      78      79      90      89       0       0       0       0
      72       1      79      80      91      90       0       0       0       0
      73       1      80      81      92      91       0       0       0       0
      74       1      81      82      93      92       0       0       0       0
      75       1      82      83      94      93       0       0       0       0
      76       1      83      84      95      94       0       0       0       0
      77       1      84      85      96      95       0       0       0       0
      78       1      85      86      97      96       0       0       0       0
      79       1      86      87      98      97       0       0       0       0
      80       1      87      88      99      98       0       0       0       0
      81       1      89      90     101     100       0       0       0       0
      82       1      90      91     102     101       0       0       0       0
      83       1      91      92     103     102       0       0       0       0
      84       1      92      93     104     103       0       0       0       0
      85       1      93      94     105     104       0       0       0       0
      86       1      94      95     106     105       0       0       0       0
      87       1      95      96     107     106       0       0       0       0
      88       1      96      97     108     107       0       0       0       0
      89       1      97      98     109     108       0       0       0       0
      90       1      98      99     110     109       0       0       0       0
      91       1     100     101     112     111       0       0       0       0
      92       1     101     102     113     112       0       0       0       0
      93       1     102     103     114     113       0       0       0       0
      94       1     103     104     115     114       0       0       0       0
      95       1     104     105     116     115       0       0       0       0
      96       1     105     106     117     116       0       0       0       0
      97       1     106     107     118     117       0       0       0       0
      98       1     107     108     119     118       0       0       0       0
      99       1     108     109     120     119       0       0       0       0
     100       1     109     110     121     120       0       0       0       0
     101       2     122     123     129     128       0       0       0       0
     102       2     123     124     130     129       0       0       0       0
     103       2     124     125     131     130       0       0       0       0
     104       2     125     126     132     131       0       0       0       0
     105       2     126     127     133     132       0       0       0       0
     106       2     128     129     135     134       0       0       0       0
     107       2     129     130     136     135       0       0       0       0
     108       2     130     131     137     136       0       0       0       0
     109       2     131     132     138     137       0       0       0       0
     110       2     132     133     139     138       0       0       0       0
     111       2     134     135     141     140       0       0       0       0
     112       2     135     136     142     141       0       0       0       0
     113       2     136     137     143     142       0       0       0       0
     114       2     137     138     144     143       0       0       0       0
     115       2     138     139     145     144       0       0       0       0
     116       2     140     141     147     146       0       0       0       0
     117       2     141     142     148     147       0       0       0       0
     118       2     142     143     149     148       0       0       0       0
     119       2     143     144     150     149       0       0       0       0
     120       2     144     145     151     150       0       0       0       0
     121       2     146     147     153     152       0       0       0       0
     122       2     147     148     154     153       0       0       0       0
     123       2     148     149     155     154       0       0       0       0
     124       2     149     150     156     155       0       0       0       0
     125       2     150     151     157     156       0       0       0       0
*NODE
$#   nid               x               y               z      tc      rc  
       1             0.0             0.0             0.0       0       0
       2             0.0             1.0             0.0       0       0
       3             0.0             2.0             0.0       0       0
       4             0.0             3.0             0.0       0       0
       5             0.0             4.0             0.0       0       0
       6             0.0             5.0             0.0       0       0
       7             0.0             6.0             0.0       0       0
       8             0.0             7.0             0.0       0       0
       9             0.0             8.0             0.0       0       0
      10             0.0             9.0             0.0       0       0
      11             0.0            10.0             0.0       0       0
      12             1.0             0.0             0.0       0       0
      13             1.0             1.0             0.0       0       0
      14             1.0             2.0             0.0       0       0
      15             1.0             3.0             0.0       0       0
      16             1.0             4.0             0.0       0       0
      17             1.0             5.0             0.0       0       0
      18             1.0             6.0             0.0       0       0
      19             1.0             7.0             0.0       0       0
      20             1.0             8.0             0.0       0       0
      21             1.0             9.0             0.0       0       0
      22             1.0            10.0             0.0       0       0
      23             2.0             0.0             0.0       0       0
      24             2.0             1.0             0.0       0       0
      25             2.0             2.0             0.0       0       0
      26             2.0             3.0             0.0       0       0
      27             2.0             4.0             0.0       0       0
      28             2.0             5.0             0.0       0       0
      29             2.0             6.0             0.0       0       0
      30             2.0             7.0             0.0       0       0
      31             2.0             8.0             0.0       0       0
      32             2.0             9.0             0.0       0       0
      33             2.0            10.0             0.0       0       0
      34             3.0             0.0             0.0       0       0
      35             3.0             1.0             0.0       0       0
      36             3.0             2.0             0.0       0       0
      37             3.0             3.0             0.0       0       0
      38             3.0             4.0             0.0       0       0
      39             3.0             5.0             0.0       0       0
      40             3.0             6.0             0.0       0       0
      41             3.0             7.0             0.0       0       0
      42             3.0             8.0             0.0       0       0
      43             3.0             9.0             0.0       0       0
      44             3.0            10.0             0.0       0       0
      45             4.0             0.0             0.0       0       0
      46             4.0             1.0             0.0       0       0
      47             4.0             2.0             0.0       0       0
      48             4.0             3.0             0.0       0       0
      49             4.0             4.0             0.0       0       0
      50             4.0             5.0             0.0       0       0
      51             4.0             6.0             0.0       0       0
      52             4.0             7.0             0.0       0       0
      53             4.0             8.0             0.0       0       0
      54             4.0             9.0             0.0       0       0
      55             4.0            10.0             0.0       0       0
      56             5.0             0.0             0.0       0       0
      57             5.0             1.0             0.0       0       0
      58             5.0             2.0             0.0       0       0
      59             5.0             3.0             0.0       0       0
      60             5.0             4.0             0.0       0       0
      61             5.0             5.0             0.0       0       0
      62             5.0             6.0             0.0       0       0
      63             5.0             7.0             0.0       0       0
      64             5.0             8.0             0.0       0       0
      65             5.0             9.0             0.0       0       0
      66             5.0            10.0             0.0       0       0
      67             6.0             0.0             0.0       0       0
      68             6.0             1.0             0.0       0       0
      69             6.0             2.0             0.0       0       0
      70             6.0             3.0             0.0       0       0
      71             6.0             4.0             0.0       0       0
      72             6.0             5.0             0.0       0       0
      73             6.0             6.0             0.0       0       0
      74             6.0             7.0             0.0       0       0
      75             6.0             8.0             0.0       0       0
      76             6.0             9.0             0.0       0       0
      77             6.0            10.0             0.0       0       0
      78             7.0             0.0             0.0       0       0
      79             7.0             1.0             0.0       0       0
      80             7.0             2.0             0.0       0       0
      81             7.0             3.0             0.0       0       0
      82             7.0             4.0             0.0       0       0
      83             7.0             5.0             0.0       0       0
      84             7.0             6.0             0.0       0       0
      85             7.0             7.0             0.0       0       0
      86             7.0             8.0             0.0       0       0
      87             7.0             9.0             0.0       0       0
      88             7.0            10.0             0.0       0       0
      89             8.0             0.0             0.0       0       0
      90             8.0             1.0             0.0       0       0
      91             8.0             2.0             0.0       0       0
      92             8.0             3.0             0.0       0       0
      93             8.0             4.0             0.0       0       0
      94             8.0             5.0             0.0       0       0
      95             8.0             6.0             0.0       0       0
      96             8.0             7.0             0.0       0       0
      97             8.0             8.0             0.0       0       0
      98             8.0             9.0             0.0       0       0
      99             8.0            10.0             0.0       0       0
     100             9.0             0.0             0.0       0       0
     101             9.0             1.0             0.0       0       0
     102             9.0             2.0             0.0       0       0
     103             9.0             3.0             0.0       0       0
     104             9.0             4.0             0.0       0       0
     105             9.0             5.0             0.0       0       0
     106             9.0             6.0             0.0       0       0
     107             9.0             7.0             0.0       0       0
     108             9.0             8.0             0.0       0       0
     109             9.0             9.0             0.0       0       0
     110             9.0            10.0             0.0       0       0
     111            10.0             0.0             0.0       0       0
     112            10.0             1.0             0.0       0       0
     113            10.0             2.0             0.0       0       0
     114            10.0             3.0             0.0       0       0
     115            10.0             4.0             0.0       0       0
     116            10.0             5.0             0.0       0       0
     117            10.0             6.0             0.0       0       0
     118            10.0             7.0             0.0       0       0
     119            10.0             8.0             0.0       0       0
     120            10.0             9.0             0.0       0       0
     121            10.0            10.0             0.0       0       0
     122             2.5             7.5             1.0       0       0
     123             2.5             6.5             1.0       0       0
     124             2.5             5.5             1.0       0       0
     125             2.5             4.5             1.0       0       0
     126             2.5             3.5             1.0       0       0
     127             2.5             2.5             1.0       0       0
     128             3.5             7.5             1.0       0       0
     129             3.5             6.5             1.0       0       0
     130             3.5             5.5             1.0       0       0
     131             3.5             4.5             1.0       0       0
     132             3.5             3.5             1.0       0       0
     133             3.5             2.5             1.0       0       0
     134             4.5             7.5             1.0       0       0
     135             4.5             6.5             1.0       0       0
     136             4.5             5.5             1.0       0       0
     137             4.5             4.5             1.0       0       0
     138             4.5             3.5             1.0       0       0
     139             4.5             2.5             1.0       0       0
     140             5.5             7.5             1.0       0       0
     141             5.5             6.5             1.0       0       0
     142             5.5             5.5             1.0       0       0
     143             5.5             4.5             1.0       0       0
     144             5.5             3.5             1.0       0       0
     145             5.5             2.5             1.0       0       0
     146             6.5             7.5             1.0       0       0
     147             6.5             6.5             1.0       0       0
     148             6.5             5.5             1.0       0       0
     149             6.5             4.5             1.0       0       0
     150             6.5             3.5             1.0       0       0
     151             6.5             2.5             1.0       0       0
     152             7.5             7.5             1.0       0       0
     153             7.5             6.5             1.0       0       0
     154             7.5             5.5             1.0       0       0
     155             7.5             4.5             1.0       0       0
     156             7.5             3.5             1.0       0       0
     157             7.5             2.5             1.0       0       0
     158            15.0             0.0             0.0       0       0
     159            15.0             1.0             0.0       0       0
     160            15.0             2.0             0.0       0       0
     161            15.0             3.0             0.0       0       0
     162            15.0             4.0             0.0       0       0
     163            15.0             5.0             0.0       0       0
     164            15.0             6.0             0.0       0       0
     165            15.0             7.0             0.0       0       0
     166            15.0             8.0             0.0       0       0
     167            15.0             9.0             0.0       0       0
     168            15.0            10.0             0.0       0       0
     169            16.0             0.0             0.0       0       0
     170            16.0             1.0             0.0       0       0
     171            16.0             2.0             0.0       0       0
     172            16.0             3.0             0.0       0       0
     173            16.0             4.0             0.0       0       0
     174            16.0             5.0             0.0       0       0
     175            16.0             6.0             0.0       0       0
     176            16.0             7.0             0.0       0       0
     177            16.0             8.0             0.0       0       0
     178            16.0             9.0             0.0       0       0
     179            16.0            10.0             0.0       0       0
     180            17.0             0.0             0.0       0       0
     181            17.0             1.0             0.0       0       0
     182            17.0             2.0             0.0       0       0
     183            17.0             3.0             0.0       0       0
     184            17.0             4.0             0.0       0       0
     185            17.0             5.0             0.0       0       0
     186            17.0             6.0             0.0       0       0
     187            17.0             7.0             0.0       0       0
     188            17.0             8.0             0.0       0       0
     189            17.0             9.0             0.0       0       0
     190            17.0            10.0             0.0       0       0
     191            18.0             0.0             0.0       0       0
     192            18.0             1.0             0.0       0       0
     193            18.0             2.0             0.0       0       0
     194            18.0             3.0             0.0       0       0
     195            18.0             4.0             0.0       0       0
     196            18.0             5.0             0.0       0       0
     197            18.0             6.0             0.0       0       0
     198            18.0             7.0             0.0       0       0
     199            18.0             8.0             0.0       0       0
     200            18.0             9.0             0.0       0       0
     201            18.0            10.0             0.0       0       0
     202            19.0             0.0             0.0       0       0
     203            19.0             1.0             0.0       0       0
     204            19.0             2.0             0.0       0       0
     205            19.0             3.0             0.0       0       0
     206            19.0             4.0             0.0       0       0
     207            19.0             5.0             0.0       0       0
     208            19.0             6.0             0.0       0       0
     209            19.0             7.0             0.0       0       0
     210            19.0             8.0             0.0       0       0
     211            19.0             9.0             0.0       0       0
     212            19.0            10.0             0.0       0       0
     213            20.0             0.0             0.0       0       0
     214            20.0             1.0             0.0       0       0
     215            20.0             2.0             0.0       0       0
     216            20.0             3.0             0.0       0       0
     217            20.0             4.0             0.0       0       0
     218            20.0             5.0             0.0       0       0
     219            20.0             6.0             0.0       0       0
     220            20.0             7.0             0.0       0       0
     221            20.0             8.0             0.0       0       0
     222            20.0             9.0             0.0       0       0
     223            20.0            10.0             0.0       0       0
     224            21.0             0.0             0.0       0       0
     225            21.0             1.0             0.0       0       0
     226            21.0             2.0             0.0       0       0
     227            21.0             3.0             0.0       0       0
     228            21.0             4.0             0.0       0       0
     229            21.0             5.0             0.0       0       0
     230            21.0             6.0             0.0       0       0
     231            21.0             7.0             0.0       0       0
     232            21.0             8.0             0.0       0       0
     233            21.0             9.0             0.0       0       0
     234            21.0            10.0             0.0       0       0
     235            22.0             0.0             0.0       0       0
     236            22.0             1.0             0.0       0       0
     237            22.0             2.0             0.0       0       0
     238            22.0             3.0             0.0       0       0
     239            22.0             4.0             0.0       0       0
     240            22.0             5.0             0.0       0       0
     241            22.0             6.0             0.0       0       0
     242            22.0             7.0             0.0       0       0
     243            22.0             8.0             0.0       0       0
     244            22.0             9.0             0.0       0       0
     245            22.0            10.0             0.0       0       0
     246            23.0             0.0             0.0       0       0
     247            23.0             1.0             0.0       0       0
     248            23.0             2.0             0.0       0       0
     249            23.0             3.0             0.0       0       0
     250            23.0             4.0             0.0       0       0
     251            23.0             5.0             0.0       0       0
     252            23.0             6.0             0.0       0       0
     253            23.0             7.0             0.0       0       0
     254            23.0             8.0             0.0       0       0
     255            23.0             9.0             0.0       0       0
     256            23.0            10.0             0.0       0       0
     257            24.0             0.0             0.0       0       0
     258            24.0             1.0             0.0       0       0
     259            24.0             2.0             0.0       0       0
     260            24.0             3.0             0.0       0       0
     261            24.0             4.0             0.0       0       0
     262            24.0             5.0             0.0       0       0
     263            24.0             6.0             0.0       0       0
     264            24.0             7.0             0.0       0       0
     265            24.0             8.0             0.0       0       0
     266            24.0             9.0             0.0       0       0
     267            24.0            10.0             0.0       0       0
     268            25.0             0.0             0.0       0       0
     269            25.0             1.0             0.0       0       0
     270            25.0             2.0             0.0       0       0
     271            25.0             3.0             0.0       0       0
     272            25.0             4.0             0.0       0       0
     273            25.0             5.0             0.0       0       0
     274            25.0             6.0             0.0       0       0
     275            25.0             7.0             0.0       0       0
     276            25.0             8.0             0.0       0       0
     277            25.0             9.0             0.0       0       0
     278            25.0            10.0             0.0       0       0
     279            17.5             7.5             1.0       0       0
     280            17.5             6.5             1.0       0       0
     281            17.5             5.5             1.0       0       0
     282            17.5             4.5             1.0       0       0
     283            17.5             3.5             1.0       0       0
     284            17.5             2.5             1.0       0       0
     285            18.5             7.5             1.0       0       0
     286            18.5             6.5             1.0       0       0
     287            18.5             5.5             1.0       0       0
     288            18.5             4.5             1.0       0       0
     289            18.5             3.5             1.0       0       0
     290            18.5             2.5             1.0       0       0
     291            19.5             7.5             1.0       0       0
     292            19.5             6.5             1.0       0       0
     293            19.5             5.5             1.0       0       0
     294            19.5             4.5             1.0       0       0
     295            19.5             3.5             1.0       0       0
     296            19.5             2.5             1.0       0       0
     297            20.5             7.5             1.0       0       0
     298            20.5             6.5             1.0       0       0
     299            20.5             5.5             1.0       0       0
     300            20.5             4.5             1.0       0       0
     301            20.5             3.5             1.0       0       0
     302            20.5             2.5             1.0       0       0
     303            21.5             7.5             1.0       0       0
     304            21.5             6.5             1.0       0       0
     305            21.5             5.5             1.0       0       0
     306            21.5             4.5             1.0       0       0
     307            21.5             3.5             1.0       0       0
     308            21.5             2.5             1.0       0       0
     309            22.5             7.5             1.0       0       0
     310            22.5             6.5             1.0       0       0
     311            22.5             5.5             1.0       0       0
     312            22.5             4.5             1.0       0       0
     313            22.5             3.5             1.0       0       0
     314            22.5             2.5             1.0       0       0
$*CONTACT_AUTOMATIC_ONE_WAY_SURFACE_TO_SURFACE_TIEBREAK_ID
*CONTACT_AUTOMATIC_SURFACE_TO_SURFACE_TIEBREAK_ID
$#     cid                                                                 title
         0Contact 1
$#    ssid      msid     sstyp     mstyp    sboxid    mboxid       spr       mpr
         2         1         3         3         0         0         0         0
$#      fs        fd        dc        vc       vdc    penchk        bt        dt
       0.0       0.0       0.0       0.0       0.0         0       0.01.00000E20
$#     sfs       sfm       sst       mst      sfst      sfmt       fsf       vsf
       1.0       1.0       0.0       0.0       1.0       1.0       1.0       1.0
$#  option      nfls      sfls     param    eraten    erates     ct2cn        cn
         8       1.0       1.0       0.4       0.0       0.0       0.0       0.0
*SET_SHELL_LIST_TITLE
down
$#     sid       da1       da2       da3       da4   
         1       0.0       0.0       0.0       0.0
$#    eid1      eid2      eid3      eid4      eid5      eid6      eid7      eid8
         1         2         3         4         5         6         7         8
         9        10        11        12        13        14        15        16
        17        18        19        20        21        22        23        24
        25        26        27        28        29        30        31        32
        33        34        35        36        37        38        39        40
        41        42        43        44        45        46        47        48
        49        50        51        52        53        54        55        56
        57        58        59        60        61        62        63        64
        65        66        67        68        69        70        71        72
        73        74        75        76        77        78        79        80
        81        82        83        84        85        86        87        88
        89        90        91        92        93        94        95        96
        97        98        99       100         0         0         0         0
*SET_SHELL_LIST_TITLE
top
$#     sid       da1       da2       da3       da4   
         2       0.0       0.0       0.0       0.0
$#    eid1      eid2      eid3      eid4      eid5      eid6      eid7      eid8
       101       102       103       104       105       106       107       108
       109       110       111       112       113       114       115       116
       117       118       119       120       121       122       123       124
       125         0         0         0         0         0         0         0
*END

**Comment by PaulAltair** [comment, marked as answer]
hi Bin, can you just zip and share the input deck as an attachment? The formatting of your model is messed up by posting it as text, I had a thought though, contact forces are only shown when they are occurring, maybe that is your issue, if you add the 2 H3D requests to your 0001.rad per the attached example, you should get the maximum of contact over time in an h3d file (but it would not be readable in Paraview...)

[LSDyna_0001.zip](https://github.com/OpenRadioss/OpenRadioss/files/12682683/LSDyna_0001.zip)

In order to modify output requests when running dyna input, you can run the model 'starter' only, and modify the generated 0001,rad before running the engine

![image](https://github.com/OpenRadioss/OpenRadioss/assets/98418895/4eab4f6b-e1c5-45b2-b589-bc2441e0cebb)

**Reply by halfwitbm** [reply]
Hi Paul, Thank you very much for the help. It's helpful. I have solved the issue. The contact force can be outputted now. 

Bin
